From aaad13c8cdda90c0d93b112a6d29e75a39e91a9a Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 19 Mar 2020 14:27:37 +0200
Subject: [PATCH] [SW-10047] pll: move the private data into c file

Move the data that does not need to be exposed from header file into
the pll.c file.

Change-Id: I1742f66ab80d21fd4b272980e0d2f41ce76a15ca
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/common/pll.c b/common/pll.c
index 56b772f..14de247 100644
--- a/common/pll.c
+++ b/common/pll.c
@@ -9,6 +9,23 @@
 #include "uart.h"
 #include "pll.h"
 
+#define HL_PLL_NR		0x100
+#define HL_PLL_NF		0x104
+#define HL_PLL_OD		0x108
+#define HL_PLL_NB		0x10c
+#define HL_PLL_CFG		0x110
+#define HL_PLL_CFG_LOCK		(1 << 4)
+#define HL_PLL_LOCK_INTR	0x128
+#define HL_PLL_DATA_CHNG	0x130
+#define HL_PLL_RST		0x134
+#define HL_PLL_DIV_FACTOR	0x200
+#define HL_PLL_DIV_FACTOR_CMD	0x220
+#define HL_PLL_DIV_SEL		0x280
+#define HL_PLL_DIV_EN		0x2a0
+#define HL_PLL_CLK_GATER	0x300
+
+#define MIN_CYCLES_PER_ITER	5
+
 static u64 cycles_per_usec = BYPASS_CYCLES_PER_USEC;
 
 void hl_pll_udelay_init(void)
diff --git a/common/pll.h b/common/pll.h
index c95bee1..d567bdb 100644
--- a/common/pll.h
+++ b/common/pll.h
@@ -8,25 +8,9 @@
 #ifndef _HL_PLL_H_
 #define _HL_PLL_H_
 
-#define HL_PLL_NR		0x100
-#define HL_PLL_NF		0x104
-#define HL_PLL_OD		0x108
-#define HL_PLL_NB		0x10c
-#define HL_PLL_CFG		0x110
-#define HL_PLL_CFG_LOCK		(1 << 4)
-#define HL_PLL_LOCK_INTR	0x128
-#define HL_PLL_DATA_CHNG	0x130
-#define HL_PLL_RST		0x134
-#define HL_PLL_DIV_FACTOR	0x200
-#define HL_PLL_DIV_FACTOR_CMD	0x220
-#define HL_PLL_DIV_SEL		0x280
-#define HL_PLL_DIV_EN		0x2a0
-#define HL_PLL_CLK_GATER	0x300
-
 #define SYS_REF_CLOCK		50000000ULL
 #define USEC_PER_SEC		1000000
 #define BYPASS_CYCLES_PER_USEC	(SYS_REF_CLOCK / USEC_PER_SEC)
-#define MIN_CYCLES_PER_ITER	5
 
 enum hl_pll_bypass_val {
 	HL_PLL_REF_CLK,
